libcontainer/container.go

```diff
@@ -127,6 +123,17 @@ type BaseContainer interface {
 	// Systemerror - System error.
 	Start(process *Process) (err error)
 
+	// StartI immediatly starts the process inside the conatiner.  Returns error if process
+	// fails to start.  It does not block waiting for a SIGCONT after start returns but
+	// sends the signal when the process has completed.
+	//
+	// errors:
+	// ContainerDestroyed - Container no longer exists,
+	// ConfigInvalid - config is invalid,
+	// ContainerPaused - Container is paused,
+	// Systemerror - System error.
```

Comment:
Probably comment for Start should be changed to describe why it's waiting on signal.